keccak-air/src/columns.rs

```diff
@@ -14,7 +16,7 @@ use crate::{NUM_ROUNDS, RATE_LIMBS, U64_LIMBS};
 /// convention of `x, y, z` order, but it has the benefit that input lists map to AIR columns in a
 /// nicer way.
 #[repr(C)]
-pub(crate) struct KeccakCols<T> {
+pub struct KeccakCols<T> {
     /// The `i`th value is set to 1 if we are in the `i`th round, otherwise 0.
     pub step_flags: [T; NUM_ROUNDS],
 
```

Comment:
We could unify the `postimage` columns with the `a_prime_prime` columns, since these are never both used in the same row. They effectively serve the same purpose, and by unifying these, we can get rid of the `.when(not_final_row)` conditions. Doing the latter may reduce the degree of the STARK from 4 to 3.

Reply:
Similarly, @CarloModicaPortfolio says we can unify `step_flags[NUM_ROUNDS-1]` with `export`.